ansible/roles/test/tasks/bgp_speaker.yml

```diff
@@ -199,6 +199,7 @@
           - fib_info='/root/bgp_speaker_route.txt'
           - ipv4={{ ipv4|default(false) }}
           - ipv6={{ ipv6|default(false) }}
+          - testbed_mtu={{ mtu|default(9114) }}
         ptf_extra_options: "--relax --debug info --log-file /tmp/bgp_speaker_test.FibTest.log"
```

Comment:
is it necessary to also have "--socket-recv-size" set?
I find without this option bgp_speaker can fail, because in it sends the second packet whose size is 4210 but expects a  packet with size equal to mtu.

Reply:
Yes, it is required. Updated the PR. Pls check